core/Types.h

```diff
@@ -677,9 +677,14 @@ struct DispatchArgs {
     const TypePtr &thisType;
     const std::shared_ptr<const SendAndBlockLink> &block;
     Loc originForUninitialized;
+    // Special option used for AndType::dispatchCall: Do not produce dispatch-related errors while evaluating the call.
+    // AndTypes only require one of their component types to handle the call, so failure is the common case -- and
+    // producing good error messages is expensive!
```

Comment:
Sounds like this comment might be confusing. I can clarify this field's semantics and the AndType details I can isolate to `AndType::dispatchCall`.

Reply:
Updated.